Formula/moto.rb

```diff
@@ -17,6 +17,8 @@ class Moto < Formula
   depends_on "rust" => :build  # for cryptography
   depends_on "python@3.9"
 
+  # install using moto[all,server]; can't do automated because of junit-xml
```

Comment:
Can you clarify what this means?

Reply:
Sound the same as we have for `semgrep`: it depends on `junit-xml` but it doesn't provide a file with source code on pypi: https://pypi.org/project/junit-xml/1.9/#files

We can do either the same here:
```ruby
  # only doing this because junit-xml source is not available in PyPI for v1.9
  resource "junit-xml" do
    url "https://github.com/kyrus/python-junit-xml.git",
        revision: "4bd08a272f059998cedf9b7779f944d49eba13a6"
  end
```

or use `junit-xml-2`: https://pypi.org/project/junit-xml-2/ (the same package but with sources). We can even make it automatically (with `brew update-python-resources`) by adding to `pypi_formula_mappings.json` 
```diff
+  "moto": {
+    "extra_packages": ["junit-xml-2"],
+    "exclude_packages": ["junit-xml"]
+  },
```

Reply:
Thanks @bayandin, I'll give junit-xml-2 a shot. Adding that substitution along with the `all` and `server` extras should allow for fully hands-off resource updates.